src/main/controller/ExerciseController.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from 'express';
+import db from '../models';
+
+export class ExerciseController {
+  getAllExercises = async (_req: Request, res: Response) => {
+    try {
+      const exercises = await db.Exercise.findAll();
+      return res.status(200).send(exercises);
+    } catch (e) {
+      return res.status(400).send({ e });
+    }
+  };
+
+  addExercise = async (req: Request, res: Response) => {
+    try {
+      const newExercise = await db.Exercise.create(req.body);
```

Comment:
We might want to get some add some sort of sanity check on `req.body`, just so that we don't accidentally corrupt the data within our db.